src/components/UI/organisms/MenuItem/MenuItem.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+:root {
+  --main: #fad390;
+  --secondary: #636e72;
+}
+
+.MenuItem {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 1fr repeat(2, 0.5fr) repeat(2, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--main);
+  align-items: center;
+  margin: 1rem;
+  padding: 1rem;
+  border: 2px solid var(--secondary);
+  border-radius: 15px;
+}
+
+.MuiTypography-root.menuItem__header {
+  grid-area: 1 / 1 / 2 / 4;
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+.menuItem__description {
+  grid-area: 2 / 1 / 3 / 4;
+}
+.menuItem__weight {
+  grid-area: 3 / 1 / 4 / 2;
+}
+.menuItem__price {
+  grid-area: 3 / 3 / 4 / 4;
+}
+.menuItem__decrease-button {
+  grid-area: 4 / 1 / 5 / 2;
+}
+.menuItem__amount {
+  grid-area: 4 / 2 / 5 / 3;
+}
+.menuItem__increase-button {
+  grid-area: 4 / 3 / 5 / 4;
+}
+.MuiButton-root.menuItem__more-info-button {
+  grid-area: 5 / 1 / 6 / 4;
+  margin-top: 1rem;
+}
+
+.menuItem__modal {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 40px;
+  border: 2px solid var(--secondary);
+  box-shadow: 24;
+  width: 400;
+  background-color: var(--main);
+}
+
+.MuiTypography-root.menuItem__modal--additional {
+  font-size: 2rem;
+  font-weight: 700;
```

Comment:
A good practice is to move the `repeatable` values of some styles to a `variables.scss` file which you `include` or `use` in other to use them - thanks to that if you change `one` variable for all elements in one place, you change them in the whole app